CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 6.0.0
+
+## Remove Parcelify Loader
+
+We no longer use Parcelify Loader in our webpack config. You should now import any static assets such as stylesheets, fonts, or icons, directly in the JavaScript files that need them.
```

Comment:
Maybe we can offer advice on how to setup the parcelify loader in their project if they want to continue using it?

Reply:
I didn't mean offer advice in the documentation on using parcelify, I meant to just mention it in the changelog as this is where people will look to see what they need to update to upgrade. We don't necessarily need to mention it in the documentation as if they're coming to carbon-factory brand new they won't necessarily know what parcelify is.